collections/535

```diff
@@ -0,0 +1,2 @@
+meta.json
```

Comment:
{"name":"535","slug":"535","supply":"1","description":"535 is a single 1/1 inscription that contains the entire set of 535 DeGods that were burned and brought back to live forever on bitcoin.","twitter_link":"https://twitter.com/535degods","discord_link":"","website_link":"","inscription_icon":""}